pkg/tuple/validation.go

```diff
@@ -58,39 +54,3 @@ func ValidateUser(tk *openfgapb.TupleKey) error {
 	}
 	return nil
 }
-
-// ValidateTuple returns whether a *openfgapb.TupleKey is valid
-func ValidateTuple(ctx context.Context, backend storage.TypeDefinitionReadBackend, store, authorizationModelID string, tk *openfgapb.TupleKey, dbCallsCounter utils.DBCallCounter) (*openfgapb.Userset, error) {
```

Comment:
I moved these because of a cyclical import issues with tuple --> storage --> tuple. This validation that depends on a reference to a storage interface probably doesn't belong in public utility methods like this, so I moved them to a validation package in the server parent package. If we prefer them living somewhere other than where I put them we can move them. But they shouldn't exist here.